service/eveapi.py

```diff
@@ -346,19 +346,6 @@ def __getattr__(self, this):
         # perform arcane attribute majick trick
         return _Context(self._root, self._path + "/" + this, self.parameters)
 
-    def __call__(self, **kw):
-        if kw:
-            # specified keywords override contextual ones
-            for k, v in self.parameters.iteritems():
-                if k not in kw:
-                    kw[k] = v
-        else:
-            # no keywords provided, just update with contextual ones.
-            kw.update(self.parameters)
-
-        # now let the root context handle it further
-        return self._root(self._path, **kw)
-
```

Comment:
> It's a little weird the way they're making it callable, but I reverted it.

How so? Checking on it, it's a method that is definitely used. I believe it's used so that you can make a call to `api.characters` and it automatically knows where to go without having to code in support for the Characters api. What was pyCharm complaining about?